core/templates/services/UpgradedServices.ts

```diff
@@ -26,6 +26,8 @@ import {
   ɵangular_packages_common_http_http_d
 } from '@angular/common/http';
 
+import { AdminBackendApiService } from
```

Comment:
You need to add all of these new services to OppiaAngularRoot as well. You can take a look at step 12 here: https://github.com/oppia/oppia/wiki/Angular-Migration#implementation-details-to-migrate-services for further instructions.

Reply:
done